src/components/ParserComponent.tsx

```diff
@@ -254,8 +260,10 @@ export default class ParserComponent extends React.Component<ParserInterface,
         };
       });
 
-      if (this.props.fileType === FileType.csv) {
+      const temp: File = fileEvent.target.files[0];
+      if (this.props.fileType === FileType.csv && this.isValid(temp)) {
         await this.parseCsv(fileEvent);
+        this.sortData(this.state.data);
```

Comment:
did you remove the console.log() in parseCsv() that calls sortData()?

Reply:
ya i removed it